lnst/Controller/ContainerPoolManager.py

```diff
@@ -99,14 +109,19 @@ def get_original_network_name(self, network: str):
     def _podman_connect(self, podman_uri: str):
         logging.debug("Connecting to Podman API")
         try:
+            from podman import PodmanClient
             client = PodmanClient(base_url=podman_uri, timeout=60)
             client.info()  # info() will try to connect to the API
+        except ModuleNotFoundError:
+            msg = f"Module podman not found, see documentation for details"
+            logging.error(msg)
+            raise PoolManagerError(msg)
         except APIError as e:
             raise PoolManagerError(f"Could not connect to Podman API: {e}")
 
         self._podman_client = client
 
-    def _check_machine(self, machine: Machine):
```

Comment:
I think you can keep this typehint. The Machine is not lazy imported.

Reply:
According to this line in the file, it comes from LNST:
```
 from lnst.Controller.Machine import Machine 
```

Reply:
Thanks, i just double checked that and realized what you meant